src/map/pc.c

```diff
@@ -10354,16 +10354,7 @@ int pc_readdb(void) {
 		}
 	}
 	fclose(fp);
-	for (i = 0; i < JOB_MAX; i++) {
-		if (!pc->db_checkid(i)) continue;
-		if (i == JOB_WEDDING || i == JOB_XMAS || i == JOB_SUMMER)
-			continue; //Classes that do not need exp tables.
-		j = pc->class2idx(i);
```

Comment:
if I'm not mistaken, now that this line is removed, the `j` variable is unused in this function?